src/zjs_modules.c

```diff
@@ -123,8 +123,40 @@ static jerry_value_t native_require_handler(const jerry_value_t function_obj,
             return mod->instance;
         }
     }
+    ZJS_PRINT("Native module not found, searching for JavaScript module %s\n", module);
```

Comment:
May just want to change this to a DBG_PRINT(). We really need to go though and change all prints to DBG_PRINTS or ERR_PRINTS unless we have a good reason for always printing.